src/commands/CustomCommands.ts

```diff
@@ -3,14 +3,20 @@ import { CustomCommand } from '../types';
 export const customCommands: Array<CustomCommand> = [
   {
     command: "runelite",
-    message: "Enabling the XP Updater plugin on Runelite is an easy, reliable way to keep your Wise Old Man profile up to date.",
+    message: "Enabling the XP Updater plugin on Runelite is a fast, easy, reliable way to keep your Wise Old Man profile up to date. See below for help enabling it.\n*(We have also created our very own plugin to do this and much more! For more information, use the `!plugin` command.)*",
     image: "https://imgur.com/tv0Jro8",
     public: true
   },
+  {
+    command: "plugin",
+    message: "We have created a plugin for Runelite that allows players to easily track progress, manage groups, see upcoming and current competitions, and much more!\nTo download, navigate to the Plugin Hub, and type “Wise Old Man” into the search bar. After installing, you will need to ensure the plugin is enabled. See below for a walkthrough."
```

Comment:
missing a comma at the end of this line.

`message: "blah blah",`

Reply:
Thanks for catching that, fixed